ui/hooks/gasFeeInput/useGasFeeInputs.test.js

```diff
@@ -37,6 +37,7 @@ jest.mock('react-redux', () => {
   return {
     ...actual,
     useSelector: jest.fn(),
+    useDispatch: () => jest.fn(),
```

Comment:
Instead of stubbing `useDispatch`, what about stubbing the background functions?

Reply:
I am not sure if stubbing background functions is better.

Reply:
Sorry, by background functions I really mean actions. But in any case, I'm just not sure if we need to be stubbing what Redux is doing under the hood. It feels to me that we can go ahead and let actions get dispatched. I suppose this would require more changes to this test file to change this, so it's okay with me. Probably worthy of a bigger discussion.